lang/lualanes/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lualanes
-PKG_VERSION:=3.13.0
-PKG_RELEASE:=2
+PKG_VERSION:=3.16.2
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/LuaLanes/lanes/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=53a17d7ee11f17ca0543ae5aa640208dcb62d37862a0d0ea450455fae12c8ff1
-PKG_BUILD_DIR:=$(BUILD_DIR)/lanes-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/LuaLanes/lanes.git
+PKG_SOURCE_VERSION:=489f7caf9ed893c1e8efc5c1b6ecb757a2131932
+PKG_MIRROR_HASH:=667042a2d773bb4d6139abca1ed2b62c2b6b3f4dd7c5446914de548625ea08f8
```

Comment:
This was changed recently. Why are we now checking out the sourcess instead of the downloading the tagged release? It does not make sense to have ``PKG_VERSION`` and ``PKG_SOURCE_VERSION`` at the same time.

Reply:
I had to switch to source because the LuaLanes release was at v3.16.2 which was outdated compared to source. There were updates in source that were needed. Those have only since been incorporated into the v3.16.3, which was only release 2 weeks ago.